doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -602,6 +602,9 @@ Matter samples
   * WPA Supplicant advanced features in all Matter samples using the :kconfig:option:`WPA_SUPP_ADVANCED_FEATURES` Kconfig option.
     This saves roughly 25 KB of FLASH memory for firmware images with Wi-Fi support.
 
+* Added ``matter_shell`` shell commands to gather the current information about the NVS settings backend.
```

Comment:
```suggestion
* Added ``matter_shell`` shell command to gather the current information about the NVS settings backend.
```
I'm guessing this should be a single command, based on the next line.

Reply:
Actually, there are multiple commands, I wrote it in different words.